core/ledger/pvtdatastorage/store.go

```diff
@@ -801,22 +810,31 @@ func (s *Store) retrieveExpiryEntries(minBlkNum, maxBlkNum uint64) ([]*expiryEnt
 
 func (s *Store) launchCollElgProc() {
 	go func() {
-		s.processCollElgEvents() // process collection eligibility events when store is opened - in case there is an unprocessed events from previous run
+		if err := s.processCollElgEvents(); err != nil {
+			// process collection eligibility events when store is opened -
+			// in case there is an unprocessed events from previous run
+			logger.Errorf("failed to process collection eligibility event [%s]", err)
+		}
```

Comment:
[%s] -> %+v (print stack trace as well). Also, it may be good to start using `logger.Errorw` that would print the error as a separate field.

Reply:
Done.